BDArmory/VesselSpawning/VesselSpawnerBase.cs

```diff
@@ -946,7 +948,7 @@ public void AddToActiveCompetition(Vessel vessel, bool airborne)
                 }
                 else if (BDArmorySettings.NO_ENGINES && SpawnUtils.CountActiveEngines(vessel) > 0) // Vessel had some active engines. Turn them off if possible.
                 {
-                    SpawnUtils.ActivateAllEngines(vessel, false);
+                    SpawnUtils.ActivateAllEngines(vessel, false); //deactivating engines when spawned landed...?
```

Comment:
My comment there was more because that no engine code originally would only run if (!airborne); hence the commenting out of the ``if (!airborne)`` condition above it.